tests/test_parallelism_framework.py

```diff
@@ -333,3 +335,28 @@ def test_InfiniteLoopingParallelismMixIn__pause__does_not_call_commands_for_each
     invoke_process_run_and_check_errors(p)
 
     assert spied_commands.call_count == 1
+
+
+def test_InfiniteLoopingParallelismMixIn__correctly_stores_time_since_initialized__in_setup_before_loop(
+    mocker,
+):
+    p = generic_infinite_looper()
+
+    expected_init_time = 15796649135715
```

Comment:
I assume the poll time is just an arbitrary number larger than an arbitrary init time, but it may be good to clarify with a comment